src/main/java/seedu/address/logic/commands/AddGuestCommand.java

```diff
@@ -18,7 +18,7 @@
 
     public static final String COMMAND_WORD = "addguest";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book. "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book.\n"
```

Comment:
Should we change to "Adds a person to the hotel"?

Reply:
Changed it to "Adds a person to Morpheus" for both since it's referring to the app!